Ccq/Console/UnitTwo.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace RLTSquare\Ccq\Console;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Psr\Log\LoggerInterface;
+use RLTSquare\Ccq\Api\Data\QueueDataInterface;
+use Magento\Framework\MessageQueue\PublisherInterface;
+
+class UnitTwo extends Command
+{
+    /**
+     *  number1
+     */
+    const FirstNumber = 'var1';
```

Comment:
Naming convention - since you opted for using constants, it would have been better to go with something like private const INPUT_OPTION_VAR1 = 'var1';. private const INPUT_OPTION_VAR2 = 'var2';. that would emphasize the meaning more clearly.